perllib/FixMyStreet/App/Form/Waste/Garden/Bexley.pm

```diff
@@ -14,4 +20,102 @@ has_page bank_details => (
     next => 'summary',
 );
 
+sub validate {
+    my $self = shift;
+
+    if ( $self->page_name eq 'bank_details' ) {
+        my $sort_code = $self->field('sort_code');
+        my $account_number = $self->field('account_number');
+        return 1 unless $sort_code && $account_number;
+        return 1 if $self->_validate_bank_details($sort_code, $account_number);
+    }
+
+    $self->next::method();
+}
+
+=head2 * _validate_bank_details
+
+Takes sort code/account number fields and validates values against the Access
+PaySuite bankchecker API which performs 'modulus checks' on them.
+
+Returns 0 if valid, 1 if invalid. If invalid, sets form errors accordingly.
+
+=cut
+
+
+sub _validate_bank_details {
+    my ($self, $sort_code, $account_number ) = @_;
+
+    # don't bother calling the API if we don't have both values
+    return 1 unless $sort_code->value && $account_number->value;
+
+    my $cfg = $self->{c}->cobrand->feature('payment_gateway');
+
+    # fail validation if not configured - we don't want to set up invalid DDs.
+    return 1 unless $cfg && $cfg->{validator_url};
```

Comment:
This didn't seem to work in my testing, it was allowing any values through. Do you need to call `add_form_error` if the URL is missing, perhaps?